packages/atomic/src/components/search/atomic-breadbox/atomic-breadbox.tsx

```diff
@@ -26,7 +26,15 @@ import {Hidden} from '../../common/stencil-hidden';
 import {Bindings} from '../atomic-search-interface/atomic-search-interface';
 
 /**
- * The `atomic-breadbox` component creates breadcrumbs that display a summary of the currently active facet values.
+ * The _Breadbox_ component helps the information seeker keep track of the navigational state of the currently active facet values, located in a single place on the search page.
+ *
+ * In most cases, these are on the top of a page between the search bar and the results.
```

Comment:
The "placement"-related information would probably be in the "Usage" section, e.g., 'This component is typically placed within the "status" section of the layout'.

See, for example, the atomic-query-summary.mdx.